raiden/tasks.py

```diff
@@ -8,12 +9,14 @@
 from gevent.queue import Queue
 import structlog
 
+
 from raiden.exceptions import RaidenShuttingDown
 from raiden.utils import get_system_spec
 
 CHECK_VERSION_INTERVAL = 3 * 60 * 60
 LATEST = 'https://api.github.com/repos/raiden-network/raiden/releases/latest'
 RELEASE_PAGE = 'https://github.com/raiden-network/raiden/releases'
+SECURITY_EXPRESSION = 'CRITICAL UPDATE[:|a-zA-Z 1-9]*.'
```

Comment:
That regex could use some work IMO.
At the very least it's missing `0` but even then it won't catch any emojis, accented characters etc.

I'd suggest to sourround the message that should be shown in the client with separating characters (e.g. `[]` , `{}`, `!!`). That will make the regex much simpler since we can use a negative match then.